src/ClientResolver.php

```diff
@@ -412,6 +422,21 @@ public static function _apply_debug($value, array &$args, HandlerList $list)
         }
     }
 
+    public static function _apply_stats($value, array &$args, HandlerList $list)
+    {
+        $defaults = array_fill_keys(
```

Comment:
The value passed in is either an associative array or a boolean. The individual collectors only default to `true` if `$value === true`, not if it's an array. In all other cases collectors default to `false`.